README.md

```diff
@@ -56,43 +57,38 @@ Some extensions are not documented. We encourage you to add documentation for th
 Installation
 ============
 
-For most installs, the `pip`-based command works for most people.
-
-For complex installation scenarios, please look at the documentation for
-installing notebook extensions, server extensions, pre/postprocessors, and
-templates at the Jupyter homepage http://www.jupyter.org.
-More information can also be found in the
-[Wiki](https://github.com/ipython-contrib/IPython-notebook-extensions/wiki).
-
-
 The install process has two steps:
 
```

Comment:
Sure, sounds reasonable. So that would then replace the current bullet points, I guess? PR away 😉